grpc/server_test.go

```diff
@@ -0,0 +1,71 @@
+package grpc
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	blog "github.com/letsencrypt/boulder/log"
+	"github.com/letsencrypt/boulder/test"
+	"google.golang.org/grpc/health"
+)
+
+func Test_serverBuilder_initLongRunningCheck(t *testing.T) {
```

Comment:
Unfortunately, because go [doesn't have good support for mocking NewTicker](https://github.com/golang/go/issues/8869#issuecomment-1480217777), I don't think there's any way to work around these Sleeps. So maybe at least mark this function with `t.Parallel()` so they don't cost us quite as much?